task_manager/Model/task.py

```diff
@@ -1,33 +0,0 @@
-from datetime import datetime
-from ..__init__ import db
-
-
-class Task(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
-    content = db.Column(db.String(200), nullable=False)
-    date_created = db.Column(db.DateTime, default=datetime.utcnow)
-
-    def __repr__(self):
-        return '<Task %r>' % self.id
-
-
-'''
-from sqlalchemy import Column, DateTime, ForeignKey, Integer, String, Text
-from ..__init__ import db
-
-
-class Task(db.Model):
-    __tablename__ = 'task'
-    id = Column(Integer, primary_key=True)
-    title = Column(String(80), unique=True, nullable=False)
```

Comment:
What do you replace this file with?

Reply:
there are two folders containing database model definitions. this was the one meghana was using but we've since started using the other model.py since this model is incomplete.